x-pack/plugins/fleet/public/applications/fleet/sections/settings/services/agent_and_policies_count.tsx

```diff
@@ -5,33 +5,59 @@
  * 2.0.
  */
 
-import { sendGetAgentPolicies, sendGetAgents } from '../../../hooks';
+import { sendGetAgentPolicies, sendGetPackagePolicies, sendGetAgents } from '../../../hooks';
 import type { Output } from '../../../types';
-import { AGENT_POLICY_SAVED_OBJECT_TYPE, SO_SEARCH_LIMIT } from '../../../constants';
+import {
+  AGENT_POLICY_SAVED_OBJECT_TYPE,
+  PACKAGE_POLICY_SAVED_OBJECT_TYPE,
+  SO_SEARCH_LIMIT,
+} from '../../../constants';
 
 export async function getAgentAndPolicyCountForOutput(output: Output) {
-  let kuery = `${AGENT_POLICY_SAVED_OBJECT_TYPE}.data_output_id:"${output.id}" or ${AGENT_POLICY_SAVED_OBJECT_TYPE}.monitoring_output_id:"${output.id}"`;
+  let agentPolicyKuery = `${AGENT_POLICY_SAVED_OBJECT_TYPE}.data_output_id:"${output.id}" or ${AGENT_POLICY_SAVED_OBJECT_TYPE}.monitoring_output_id:"${output.id}"`;
+  const packagePolicyKuery = `${PACKAGE_POLICY_SAVED_OBJECT_TYPE}.output_id:"${output.id}"`;
+
   if (output.is_default) {
-    kuery += ` or (not ${AGENT_POLICY_SAVED_OBJECT_TYPE}.data_output_id:*)`;
+    agentPolicyKuery += ` or (not ${AGENT_POLICY_SAVED_OBJECT_TYPE}.data_output_id:*)`;
   }
+
   const agentPolicies = await sendGetAgentPolicies({
-    kuery,
+    kuery: agentPolicyKuery,
     page: 1,
     perPage: SO_SEARCH_LIMIT,
   });
 
   if (agentPolicies.error) {
     throw agentPolicies.error;
   }
-  const agentPolicyCount = agentPolicies.data?.items?.length ?? 0;
+
+  const packagePolicies = await sendGetPackagePolicies({
```

Comment:
Not necessarily related to that PR but I have more and more concerns about retrieving all package policies client side, this will be a blocker to support a larger number of policies, and I am not sure of the value of showing an exact count of policies for the user. 
I am wondering if this could be simplified to two scenarios, the output is not used by any agent policy or the output is used by at least one agent policy.

Reply:
I agree with this, and I actually saw that on some of our other settings confirmation modals, we don't show the exact count like this. we just tell the user that this will agents enrolled into affected policies. I'll capture this in a follow up enhancement to remove these granular counts in modals

Reply:
Could some one explain what this block is referring to? is it referring to the counts in the Fleet table?

Reply:
@nimarezainia It is the count in the confirmation modal when user updates/deletes an output, like "this will update X policies used by X agents"

Reply:
@nimarezainia @nchaulet I opened https://github.com/elastic/kibana/issues/190432